src/main/groovy/4configureEc2Plugin.groovy

```diff
@@ -0,0 +1,175 @@
+/**
+*
+* Configure the EC2 plugin
+*
+**/
+import java.util.logging.Logger
+import jenkins.model.Jenkins
+import hudson.model.Node
+import hudson.plugins.ec2.*
+import com.amazonaws.services.ec2.model.InstanceType
+
+@Grapes([
+    @Grab(group='org.yaml', module='snakeyaml', version='1.17')
+])
+import org.yaml.snakeyaml.Yaml
+
+Logger logger = Logger.getLogger("")
+Jenkins jenkins = Jenkins.getInstance()
+Yaml yaml = new Yaml()
+
+String configPath = System.getenv("JENKINS_CONFIG_PATH")
+try {
+    configText = new File("${configPath}/ec2_config.yml").text
+} catch (FileNotFoundException e) {
+    logger.severe("Cannot find config file path @ ${configPath}/ec2_config.yml")
+    jenkins.doSafeExit(null)
+    System.exit(1)
+}
+ec2Config = yaml.load(configText)
+ec2CloudConfig = ec2Config.CLOUDS
```

Comment:
I think we should type as often as possible. It makes scripts clearer, and adds another layer of protection, whereas the inference typing in groovy can obfuscate issues with the data we pass in